apollos-church-api/src/data/ActionAlgorithms.js

```diff
@@ -68,23 +77,25 @@ class dataSource extends ActionAlgorithm.dataSource {
   async weeklyScriptureFeedAlgorithm({
     category = '',
     channelIds = [],
-    limit = 5,
+    limit = 50,
     skip = 0,
   } = {}) {
     const { ContentItem } = this.context.dataSources;
 
     const items = (await Promise.all(
       channelIds.map(async (channel) =>
         (await ContentItem.byContentChannelId(channel, category, false))
+          .sort([{ field: 'StartDateTime', direction: 'asc' }])
```

Comment:
@conrad-vanl I get this, but one more question that I have is, this `.sort` happens before the `.top(limit)`. Doesn't that mean that using OData on the rock api, I will be sorting all the items by newest, then I run the limit which will be the top 50 items. Therefore, it will always be the 50 most recent items. Or is there something else going on here where the order these are called doesn't matter?